CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.12.0
+
+Blank line parsing: parse new lines and maintain number of new lines in markdown
+instead of treating them as separtors.
+
+`===` separators for front matter.
```

Comment:
While pre `1.0` we've been using `Minor` bumps as equivalents to `Major`, certainly we've done the same for other syntax breaking changes in the past, and i'm happy with that. Once we do hit `1.0` then we'll follow semver properly.